Sources/Apollo/ApolloStore.swift

```diff
@@ -141,17 +141,17 @@ public final class ApolloStore {
       self.cacheKeyForObject = cacheKeyForObject
     }
     
-    public func readObject<Query: GraphQLQuery>(forQuery query: Query) throws -> JSONObject {
-      return try readObject(forSelectionSet: Query.selectionSet, withKey: rootKey(forOperation: query), variables: query.variables)
+    public func read<Query: GraphQLQuery>(query: Query) throws -> Query.Data {
```

Comment:
Sure, but there's also `readFragment`, right? Shouldn't that be `read(fragment: )` then?